packages/mobile/src/app/actions.ts

```diff
@@ -26,6 +26,7 @@ export enum Actions {
   UNLOCK = 'APP/UNLOCK',
   SET_SESSION_ID = 'SET_SESSION_ID',
   OPEN_URL = 'APP/OPEN_URL',
+  SET_REQUIRED_VERSION = 'APP/SET_REQUIRED_VERSION',
```

Comment:
Would be nice to avoid introducing new setters and try to gradually migrate to events.
How about we use `MIN_APP_VERSION_DETERMINED` instead?